Added more description and another example for Thrust Curve tuning (#871)
PX4/PX4-user_guide: en/config_mc/pid_tuning_guide_multicopter.md

### Thrust Curve

The tuning above optimises performance around the hover throttle.
But you may start to see oscillations when going towards full throttle.

To counteract that, adjust the **thrust curve** with the [THR_MDL_FAC](../advanced_config/parameter_reference.md#THR_MDL_FAC) parameter.

> **Note** The rate controller might need to be re-tuned if you change this parameter.

The mapping from motor control signals (e.g. PWM) to expected thrust is linear by default — setting `THR_MDL_FAC` to 1 makes it quadratic.
Values in between use a linear interpolation of the two. Typical values are between 0.3 and 0.5.

> If you have a [thrust stand](https://www.rcbenchmark.com/pages/series-1580-thrust-stand-dynamometer) (or can otherwise _measure_ thrust and PWM commands simultaneously), you can determine the relationship between the PWM control signal and the motor's actual thrust, and fit a function to the data.

Reviewer: ```suggestion
If you have a [thrust stand](https://www.rcbenchmark.com/pages/series-1580-thrust-stand-dynamometer) (or can otherwise _measure_ thrust and PWM commands simultaneously), you can determine the relationship between the motor control signal and the motor's actual thrust, and fit a function to the data. The motor command in PX4 called `actuator_output` can be PWM, Dshot, UAVCAN commands for the respective ESCs in  use. 
```

Author: Sounds good to me

[This Notebook][THR_MDL_FAC_Calculation] shows one way for how the thrust model factor `THR_MDL_FAC` may be calculated from previously measured thrust and PWM data.

> The curves shown in this plot are parametrized by both &alpha; and k, and also show thrust and PWM in real units (kgf and &mu;s).

Reviewer 2: ```suggestion
The curves shown in this plot are parametrized by both &alpha (the normalization factor); and k (the `THR_MDL_FAC`), and also show thrust and PWM in real units (kgf and &mu;s).
```

> However, note that in order to calculate `THR_MDL_FAC`, you will need to normalize the data betweeon 0 and 1. 

Reviewer 2: ```suggestion
In order to simplify the curve fit problem, you can normalize the data between 0 and 1 to find `k` without having to estimate &alpha (&alpha = 1 when the data is normalized).
```
